great_expectations/rule_based_profiler/types/parameter_container.py

```diff
@@ -630,6 +630,12 @@ def _get_parameter_node_attribute_names_as_lists(
 def _get_parameter_name_parts_up_to_including_reserved_literal(
     attribute_name_as_list: List[str],
 ) -> List[str]:
+    if attribute_name_as_list[0] == PARAMETER_NAME_ROOT_FOR_VARIABLES:
+        return [
+            PARAMETER_NAME_ROOT_FOR_VARIABLES,
```

Comment:
Feels a tiny bit clunky, but I understand that this is the result of the nature of `ParameterContainers`